kudos-services/src/main/java/org/exoplatform/kudos/listener/analytics/KudosSentListener.java

```diff
@@ -116,7 +118,7 @@ private void addEventStatistic(Kudos kudos) {
 
     statisticData.setModule("social");
     statisticData.setSubModule("kudos");
-    statisticData.setOperation("sendKudos");
+    statisticData.setOperation("sendKudosFrom" + source);
```

Comment:
```suggestion
    statisticData.setOperation("sendKudos");
```

Knowing that entityType already exists, it would be better to avoid changing operation name (which is more generic)

Reply:
Done!